src/CharityVaultFactory.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.6;
+
+import {ERC20} from "solmate/erc20/ERC20.sol";
+import {Bytes32AddressLib} from "./libraries/Bytes32AddressLib.sol";
+import "./CharityVault.sol";
```

Comment:
oh can we make this a named import like above pls dont like leaking other contracts into scope